client/locations-list-map.js

```diff
@@ -231,7 +260,53 @@ function multilineStringToNodes(input) {
   return returnedNodes.slice(0, -1);
 }
 
-function addMarkerContent(orgType, address, name, instructions, accepting, openBox, separator) {
+function createMakerMarkerContent(entry, separator) {
+  // Text to go into InfoWindow
+  const contentTags = separator ? [ce('h5', 'separator', ctn(entry.name))] : [ce('h5', null, ctn(entry.name))];
```

Comment:
Cleaner to put the conditional inside the call to ce, since the other two params are the same
```javascript
const contentTags = [ce('h5', separator ? 'separator' : null, ctn(entry.name))];

Reply:
Yeah that is much better. Was copy pasta code. Fixed.